plugins/woocommerce/readme.txt

```diff
@@ -362,6 +371,7 @@ WooCommerce comes with some sample data you can use to see how products look; im
 * Update - Update the Catalog searching, average rating, and product button blocks to make sure they render with the wp-block-<block-name> class. [#51617](https://github.com/woocommerce/woocommerce/pull/51617)
 * Update - Use Block Hooks to insert Account Creation block into templates automatically. [#51582](https://github.com/woocommerce/woocommerce/pull/51582)
 * Update - [ Experimental ] Moved experimental product dataviews menu to Product top menu. [#51511](https://github.com/woocommerce/woocommerce/pull/51511)
+* Dev - Checkout: Revert #52044 Add loading placeholder and payment method toggle. [#53085](https://github.com/woocommerce/woocommerce/pull/53085)
```

Comment:
✍🏼 Note: we are adding 10 lines here, but are removing 13 changelog files. That's because three of those are files are comments (ie, no changelog entry required):

- 53051-fix-52990-use-payment-method-instead-of-intent
- 53192-update-refactor-merged-packages-activation
- cherry-pick-51540-remove-changelog-entry